packages/frontend/remix/src/components/common/PlatCell/PlatDateFormatter.test.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, expect, test } from "vitest";
+
+import { PlatDateFormatter } from "./PlatDateFormatter";
+
+describe("相対時間の計算 PlatDateFormatter", () => {
+  test("不正な引数：タイムスタンプが現在時刻を超過", () => {
+    expect(() => {
+      PlatDateFormatter({
+        platTimestamp: new Date("2021-01-30T00:00:55+09:00"),
+        currentTime: new Date("2021-01-30T00:00:54+09:00"),
+      });
+    }).toThrow("Invalid time difference: platTimestamp is later than currentTime.");
+  });
+
+  test("秒差", () => {
+    expect(
+      PlatDateFormatter({
+        platTimestamp: new Date("2021-01-30T00:00:55+09:00"),
```

Comment:
テストケースが 55秒 だと視認性が悪いので、すべての数字に関して、できるだけキリのいい数字の方が好ましいですね。

たとえばですが

```
29
30
```

と

```
54
55
```

でそれぞれ並んでいるとき、前者は十の位も変化しているので視認性が高いのに対し、後者は1の位しか変化していないので見づらい、というのが視認性の良い悪いの感覚になります。

```suggestion
        platTimestamp: new Date("2020-01-01T00:00:00+09:00"),
```

※全部 00だと不都合あるケースあると思うので適宜調整してください。

---

もちろん、55秒に何かしらの意図があるのであれば、このままで問題ありません。

その場合はファイルの頭にコメント書いといてもらえるとありがたいです！